notify/notify.yaml

```diff
@@ -410,7 +412,9 @@ components:
             cursors:
               type: object
               properties:
-                $ref: '#/components/schemas/after'
+                after:
```

Comment:
same here, keep the '#/components/schemas/after' and change it one layer down

Reply:
yup!